test/unix/test_lwt_bytes.ml

```diff
@@ -530,4 +530,43 @@ let suite = suite "lwt_bytes" [
       >>= fun () ->
       Lwt.return check
     end;
+
+    test "bytes recv" begin fun () ->
+      let hostname = "127.0.0.1" in
+      let port = 20000 in
+      let get_socket_infos () =
+             Lwt_unix.gethostbyname hostname
+             >>= fun entry ->
+             let addr = Array.get entry.Unix.h_addr_list 0 in
+             let sock = Lwt_unix.socket Lwt_unix.PF_INET Lwt_unix.SOCK_STREAM 0 in
+             let sockaddr = Lwt_unix.ADDR_INET (addr, port) in
+             Lwt.return (sock, sockaddr)
```

Comment:
It's odd to pair the socket and the address. You should have one address that is shared between the client and server, but two different sockets. so that suggests one setup of `sockaddr` and two calls to `socket` at the appropriate places.